core/cmd.go

```diff
@@ -57,6 +57,22 @@ func (ch *CommandHandler) GetAppVersion() string {
 	return AppVersion.Original()
 }
 
+func (ch *CommandHandler) CheckForUpdate() bool {
+	latestVersion, err := ch.browser.GetLatestAppVersion()
+	if err != nil {
+		log.Println(err)
+		return false
```

Comment:
What if an error was cause because something was wrong with the browser itself? Not local to this function but we should maybe return an error here instead